app/lib/Plugins/SearchEngine/Elastic8.php

```diff
@@ -761,4 +761,28 @@ public function quickSearch($pn_table_num, $ps_search, $pa_options = []): array
 	public function isReindexing(): bool {
 		return (defined('__CollectiveAccess_IS_REINDEXING__') && __CollectiveAccess_IS_REINDEXING__);
 	}
+
+	/**
+	 * @throws AuthenticationException
+	 * @throws ClientResponseException
+	 * @throws ServerResponseException
+	 */
+	public function info() {
+		return $this->getClient()->info();
+	}
+
+	/**
+	 * @throws AuthenticationException
+	 * @throws ClientResponseException
+	 * @throws MissingParameterException
+	 * @throws ServerResponseException
+	 */
+	public function checkIndexes() {
+		$mapping = new Elastic8\Mapping();
+		$tables = $mapping->getTables();
+
+		return $this->getClient()->indices()->exists([
```

Comment:
I think `exists` requires an additional parameter ` 'ignore_missing' => true` in order to not throw an exception if it doesn't exist :confused:

Reply:
returning as bool correctly flags error when a single index has been deleted.